lib/linguist/languages.yml

```diff
@@ -2371,6 +2371,16 @@ JavaScript:
   interpreters:
   - node
   language_id: 183
+JavaScript+ERB:
+  type: programming
+  tm_scope: none
+  group: JavaScript
+  extensions:
+  - ".js.erb"
+  ace_mode: text
+  codemirror_mode: htmlembedded
```

Comment:
The [HTML Embedded](https://codemirror.net/mode/htmlembedded/index.html) page mentions that it supports the `x-erb` MIME type, but did you check how the actual highlighting looks like? Wouldn't JavaScript be a better fit here as well?